src/tools/debugger/FrontendConnector.java

```diff
@@ -207,18 +209,53 @@ public void awaitClient() {
   private static Map<SFarReference, String> createActorMap(
```

Comment:
Ok, would be great if you could clarify that in the code by adapting the naming of the methods, maps, and perhaps add a one line JavaDoc.